SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -14,10 +14,10 @@ Plot {
 
 		StartUp.add {
 			GUI.skin.put(\plot, (
-				gridColorX: QtGUI.palette.highlight,
-				gridColorY: QtGUI.palette.highlight,
-				fontColor: QtGUI.palette.windowText,
-				plotColor: [QtGUI.palette.windowText, Color.blue.valueBlend(QtGUI.palette.windowText), Color.red.valueBlend(QtGUI.palette.windowText), Color.green(0.7).valueBlend(QtGUI.palette.windowText)],
+				gridColorX: QtGUI.palette.highlight.sat_(0.2).val_(0.9),
+				gridColorY: QtGUI.palette.highlight.sat_(0.2).val_(0.9),
```

Comment:
@mtmccrea -- are these methods from an extension or something?  they are not in vanilla SC, and that appears to be why the unit tests are failing. i get this message on startup:

```
ERROR: Message 'sat_' not understood.
RECEIVER:
Instance of Color {    (0x115a482b8, gc=6C, fmt=00, flg=00, set=02)
  instance variables [4]
    red : Float 0.298039   13131313 3FD31313
    green : Float 0.498039   DFDFDFE0 3FDFDFDF
    blue : Float 0.749020   F7F7F7F8 3FE7F7F7
    alpha : Float 1.000000   00000000 3FF00000
}
ARGS:
   Float 0.200000   9999999A 3FC99999
CALL STACK:
	DoesNotUnderstandError:reportError
		arg this = <instance of DoesNotUnderstandError>
	Nil:handleError
		arg this = nil
		arg error = <instance of DoesNotUnderstandError>
	Thread:handleError
		arg this = <instance of Thread>
		arg error = <instance of DoesNotUnderstandError>
	Object:throw
		arg this = <instance of DoesNotUnderstandError>
	Object:doesNotUnderstand
		arg this = <instance of Color>
		arg selector = 'sat_'
		arg args = [*1]
	< closed FunctionDef >  (no arguments or variables)
	Function:doOnStartUp
		arg this = <instance of Function>
	ArrayedCollection:do
		arg this = [*6]
		arg function = <instance of Function>
		var i = 4
	List:do
		arg this = <instance of List>
		arg function = <instance of Function>
	Meta_StartUp:run
		arg this = <instance of Meta_StartUp>
	Main:startup
		arg this = <instance of Main>
		var didWarnOverwrite = false
^^ The preceding error dump is for ERROR: Message 'sat_' not understood.
RECEIVER: Color(0.29803921568627, 0.49803921568627, 0.74901960784314)
```

Reply:
`.sat_` seems to come from the `wslib` quark...

Reply:
got it. good lesson for everyone, don't do development work with extensions enabled :)

Reply:
ahhh! thanks for catching this @brianlheim (and  @dyfer)!
strange that this wasn't the error thrown in the test. I'll update the functionality with something built-in.

Reply:
well, this code isn't exercised in the test, that's the problem